doc/release-notes/9434-app-container.md

```diff
@@ -0,0 +1,5 @@
+Developers can experiment with running Dataverse in Docker: https://preview.guides.gdcc.io/en/develop/container/app-image.html
+
+This is an image developers build locally. It is not meant for production use!
+
+To provide a complete container-based local development environment, developers can deploy a Dataverse container from the new image in addition to other containers for the necessary dependencies: https://preview.guides.gdcc.io/en/develop/container/dev-usage.html
```

Comment:
I'd rather just point to the relevant extensive documentation and remove the sentence summarizing the approach. Without domain knowledge this section would confuse me.

Reply:
Note that this is a snippet for the release docs. So it's important to give some context when including this in the next release.

Reply:
You are right, I did not consider that this is part of the release note.